patrickf949/trainhub
# Enables users add and view contacts

# Description

Enables users add and view contacts.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.


# How Has This Been Tested?

1. clone the repo
`git clone https://github.com/patrickf949/trainhub.git`

2. check into the directory and checkout this branch
`cd trainhub && git checkout ft-contacts` 

3. install dependencies
` npm install `

4. run the development environment
` npm run dev`

5. open in your browser `localhost:3000/contacts`
6. try to add a contact

# Checklist:
